core/src/main/java/net/opentsdb/pools/ByteArrayPool.java

```diff
@@ -50,6 +50,7 @@ public String type() {
         .setAllocator(this)
         .setInitialCount(tsdb.getConfig().getInt(configKey(COUNT_KEY, TYPE)))
         .setMaxCount(tsdb.getConfig().getInt(configKey(COUNT_KEY, TYPE)))
+        .setArrayLength(tsdb.getConfig().getInt(configKey(LENGTH_KEY, TYPE)))
```

Comment:
Is this fixing the default length of byte array to 8K

Reply:
yup